addons/containerd/1.3.7/install.sh

```diff
@@ -179,3 +179,7 @@ function containerd_migrate_from_docker() {
     sed -i "s@\(KUBELET_KUBEADM_ARGS=\".*\)\"@\1 $containerdFlags\" @" /var/lib/kubelet/kubeadm-flags.env
     systemctl daemon-reload
 }
+
+function containerd_preflight() {
```

Comment:
not needed anymore, the `host-preflight.yaml` file will be automatically detected